src/index.js

```diff
@@ -1,6 +1,21 @@
+const Graceful = require('node-graceful');
 const { backup } = require('./backup');
 const { restore } = require('./restore');
 
+Graceful.timeout = 5000;
+Graceful.exitOnDouble = true;
+Graceful.captureExceptions = false;
```

Comment:
As we have a lot of promises, we should maybe also add `Graceful.captureRejections = true;`

Reply:
Thanks, done.